src/workerd/jsg/setup.c++

```diff
@@ -121,6 +122,7 @@ V8System::V8System(kj::Own<v8::Platform> platformParam, kj::ArrayPtr<const kj::S
 
   v8::V8::InitializePlatform(platform.get());
   v8::V8::Initialize();
+  cppgc::InitializeProcess(platform->GetPageAllocator());
```

Comment:
Do we need a corresponding `cppgc::ShutdownProcess()` when cleaning up the `V8System`?

Reply:
Oh yeah I suppose. Of course, in practice we never actually destroy `V8System` objects.